minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java

```diff
@@ -192,6 +211,20 @@ public void cleanupUnderfs() throws IOException {
   }
 
   /**
+   * Get the externally resolvable address of the master (used by unit test only).
```

Comment:
Since we probably need to change this all over the codebase, could we address it in another ticket?